lib/benchee/benchmark.ex

```diff
@@ -149,16 +165,20 @@ defmodule Benchee.Benchmark do
     end
   end
 
-  defp do_benchmark(finish_time, function, input, run_times, n, now)
-  defp do_benchmark(finish_time, _, _, run_times, _n, now)
+  defp do_benchmark(finish_time, function, input, run_times, memory_usages, n, now)
+  defp do_benchmark(finish_time, _, _, run_times, memory_usages, _n, now)
        when now > finish_time do
-    Enum.reverse run_times # restore correct order important for graphing
+    {Enum.reverse(run_times), memory_usages}
   end
-  defp do_benchmark(finish_time, function, input, run_times, n, _now) do
+  defp do_benchmark(finish_time, function, input, run_times, memory_usages, n, _now) do
+    :erlang.garbage_collect()
+    before_run = :erlang.memory(:total)
```

Comment:
I'm not too sure, but just trying it out in `iex` it seems almost like a random number generator:

```
iex(1)> :erlang.memory(:total)
18771424
iex(2)> :erlang.memory(:total)
18996328
iex(3)> :erlang.memory(:total)
19028888
iex(4)> :erlang.memory(:total)
19030472
iex(5)> :erlang.memory(:total)
19019176
iex(6)> :erlang.memory(:total)
18967328
iex(7)> :erlang.memory(:total)
18989816
```